.storybook/foundations/down-state.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta, Story } from '@storybook/blocks';
+import * as ButtonStories from '../../components/button/stories/button.stories.js';
+import * as CheckboxStories from '../../components/checkbox/stories/checkbox.stories.js';
+
+<Meta title="Foundations/Down state" />
+
+<h1>Down state</h1>
+
+Down state is a Spectrum 2 (S2) feature that creates the illusion of components being pressed-in when active. This functionality is already included in S2 components that require down state in this project. It is implemented with a CSS transform. The implementation depends on the size of the interactable element:
+
+  - If the interactable element has a width of 24px or less, we use the minimum perspective token:
+    ```
+    transform: perspective(var(--spectrum-component-size-minimum-perspective-down)) translateZ(var(--spectrum-component-size-difference-down));
+    ```
+  - If the interactable element has a width that is greater than 24px, we use the max formula:
+    ```
+    transform:
+      perspective(max(var(--spectrum-downstate-height), var(--spectrum-downstate-width) * var(--spectrum-component-size-width-ratio-down)))
+      translateZ(var(--spectrum-component-size-difference-down));
+    ```
+    - Note that in this case, users are required to develop an implementation to determine the width and height of the component. Assign these values to the `--spectrum-downstate-width` and `--spectrum-downstate-height` custom properties in a `style` attribute on the HTML element to expose them for use in the CSS.
+
+<Story of={CheckboxStories.Default} />
+<Story of={ButtonStories.DownStateDocs} />
```

Comment:
There's an outstanding issue with these where the global styles don't load unless you first go to a component page, so the buttons look wonky. Not sure yet if that will be a fix in this PR or a separate, more global one

Reply:
Was this a known issue before or something you discovered for the first time adding this story? I noticed this when creating a new story for the Checkboxes and Action Buttons in Foundations for corner-rounding as well.

Reply:
I think it has to do with the existing setup and where we're importing global tokens (probably in preview.js). It can probably be its own Jira card to work on.